debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsSnapshotProducerIT.java

```diff
@@ -58,15 +58,16 @@ public void shouldGenerateSnapshotsForDefaultDatatypes() throws Exception {
         snapshotProducer = new RecordsSnapshotProducer(context, new SourceInfo(TestHelper.TEST_SERVER), false);
 
         TestHelper.executeDDL("postgres_create_tables.ddl");
-        TestConsumer consumer = testConsumer(ALL_STMTS.size());
+        TestHelper.executeDDL("postgis_create_tables.ddl");
+        TestConsumer consumer = testConsumer(ALL_STMTS.size(), "public", "Quoted_\"");
 
         //insert data for each of different supported types
         String statementsBuilder = ALL_STMTS.stream().collect(Collectors.joining(";" + System.lineSeparator())) + ";";
         TestHelper.execute(statementsBuilder);
 
         //then start the producer and validate all records are there
         snapshotProducer.start(consumer);
-        consumer.await(TestHelper.waitTimeForRecords(), TimeUnit.SECONDS);
+        consumer.await(TestHelper.waitTimeForRecords() * 30, TimeUnit.SECONDS);
```

Comment:
Is this a leftover?

Reply:
No... when the postgis extension is loaded the `spatial_ref_sys` table is populated with ~5K rows... which takes >2s to process. The included data changes with each release so I ignore it through the Debezium unit tests.